Jenkinsfile

```diff
@@ -1,91 +1,67 @@
-// Setup milestone to stop previous build from running when a new one is launched
-// The result would be:
-//  Build 1 runs and creates milestone 1
-//  While build 1 is running, suppose build 2 fires. It has milestone 1 and milestone 2. It passes 1, which causes build #1 to abort
+@Library('jenkins-pipeline-shared-libraries')_
 
-def buildNumber = env.BUILD_NUMBER as int
-if (buildNumber > 1) milestone(buildNumber - 1)
-milestone(buildNumber)
+def changeAuthor = env.ghprbPullAuthorLogin ?: CHANGE_AUTHOR
```

Comment:
moving to RHBA ?

Was this taken from https://github.com/kiegroup/kogito-cloud-operator/pull/392/files ?

Reply:
No, this is on the aws jenkins only, we are just changing the mechanism the same as we've on the RHBA. Multibranch doesn't have to run a job only on comment, but we've that on ghprb . So for checking out the PR we're using these params and functions on the shared library

Reply:
oh I see.